src/smc-webapp/chat/REAME.md

```diff
@@ -0,0 +1,48 @@
+# Chat layout
+## Usage
+Only import from `main.coffee`.
+You should not need to import from any other file.
+
+It allows you to:
+- Initialize stores and actions for a chat
+- Render that chat
+- Destroy the same stores and actions
+
+## Overview
+CoCalc has several chat view considerations.
+- Side chat associated with files
+- Primary chat rooms (also a file)
+- Mobile and Desktop versions of both
+
+The constricting factors are primarily keyboard related or screen size related.
+ie. You cannot use certain hotkeys without a physical keyboard and certain things don't fit well on a smaller screen.
+
+opening side chat on mobile should "slide over" the main document. Basically, don't try to split the screen.
```

Comment:
I wonder if we should start saying “mobile phone”, since laptops and iPads are mobile, but have very different constraints.   Don’t change this – just a comment.